src/components/Combobox/ControlledCombobox.js

```diff
@@ -44,6 +44,10 @@ export const propTypes = {
    * whether the input is disabled
    */
   disabled: PropTypes.bool,
+  /**
+   * whether the dropdown list and icon are visible
+   */
+  hideDropdown: PropTypes.bool,
```

Comment:
Wouldn't `isStatic` or `noSearch` convey this better than `hideDropdown?`

Reply:
`noSearch` sounds good!